middleware/profiler.go

```diff
@@ -11,13 +10,13 @@ import (
 
 // Profiler is a convenient subrouter used for mounting net/http/pprof. ie.
 //
-//  func MyService() http.Handler {
-//    r := chi.NewRouter()
-//    // ..middlewares
-//    r.Mount("/debug", middleware.Profiler())
-//    // ..routes
-//    return r
-//  }
+//	func MyService() http.Handler {
+//	  r := chi.NewRouter()
+//	  // ..middlewares
+//	  r.Mount("/debug", middleware.Profiler())
+//	  // ..routes
+//	  return r
+//	}
```

Comment:
Let's use TAB instead of whitespace:

```suggestion
 //	func MyService() http.Handler {
//		r := chi.NewRouter()
//		// ..middlewares
//		r.Mount("/debug", middleware.Profiler())
//		// ..routes
//		return r
//	}
```

Reply:
Done :heavy_check_mark: Switched to tabs